sites/INFN-CLOUD-CNAF.yaml

```diff
@@ -1,6 +1,6 @@
 endpoint: https://cloud-api-pub.cr.cnaf.infn.it:5000/v3/
 gocdb: INFN-CLOUD-CNAF
-protocol: openid
+region: sdds
```

Comment:
So I lack all the background but this changes looks a bit weird to me, but maybe it's just normal.

Reply:
The protocol `openid` is the default, so was redundant information.
The actual change is adding the region in there

Reply:
BTW, this information is actually also available in GOC, see https://goc.egi.eu/portal/index.php?Page_Type=Site&id=2745, but reading it from there was more complicated than this change at the moment